.buildkite/gen-pipeline.sh

```diff
@@ -135,6 +135,13 @@ run_all() {
     ":muscle: Test MXNet MNIST (${test})" \
     "bash -c \"OMP_NUM_THREADS=1 \\\$(cat /mpirun_command) python /horovod/examples/mxnet_mnist.py\""
 
+  if [[ ${test} == *"openmpi"* ]]; then
```

Comment:
Let's run this under the "latest framework" check below, since this is not dependent on MXNet or other version.

Reply:
Done